CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 ## x.y.z (unreleased)
 
 ### Added
+- Added setting for putting the media query style block to <body> vs <head>
```

Comment:
I propose we simplify/shorten this to: "Automatically the STYLE block into the BODY". What do you think=